### ISSUE TITLE
qisrc maintainers --list bug with utf-8

### ISSUE THREAD

**Issue by alkino** [aldebaran/qibuild#44]
Hi o/

I do qisrc maintainers --list and I got 
```
Maintainers of foo/bar
[ERROR]: UnicodeEncodeError 'ascii' codec can't encode character u'\xe9' in position 1: ordinal not in range(128)
```

In qiproject.xml
```xml
<maintainer email="rdupont@example.com">Rémi Dupont</maintainer>
```

**Comment by alkino** [comment]
Bump?

**Comment by dmerejkowsky** [comment]
Closed with: b12ff7b998abbc3974c740cd183f7be10a3fafa
